_posts/2018-07-17-client-openings.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+author: chris
+title: Let us help!
+image: 'posts/diagram.jpg'
+---
+
+Apsis Labs is about to wrap up our largest contract yet, and has openings for new clients! [Contact us](mailto:contact@apsis.io) to tell us about your project.
+
+Apsis is a small custom software consultancy. That means we help you use software to solve your business problems. We specialize in mobile and web applications in the healthtech, business workflow, and game industries. We're flexible, fast, and easy to work with.
+
+We'd love to build something for you. Got this far and still not sure you need us? Here are some good clues we could help you out:
+
+ * You've got a great app idea for your business or soon-to-be business, but no idea how to turn the idea into a reality.
```

Comment:
I'd add (or, potentially replace) one of these bullet points with something about healthcare specifically.

Something like (feel free to use the example, but also feel totally free to re-write):

* You have a healthcare website that you want to build, but you aren't sure how to make sure it's technically compliant with HIPAA.